kernel-macro/Cargo.toml

```diff
@@ -15,4 +15,5 @@ proc-macro = true
 
 [dependencies]
 syn = { version = "2.0", features = ["full"]}
-quote = "1.0"
\ No newline at end of file
+quote = "1.0"
+tezos-smart-rollup-core = "0.2.0"
```

Comment:
I don't know if this dependency is required or not :thinking: IMO it's required in the examples/project not in the macro, can you check this?

Reply:
fix at this [commit](3d56b448ce27ff2ee25b9e46e3b86aecbbb74896)